test/rubocop/cop/minitest/assert_with_two_arguments_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AssertWithTwoArgumentsTest < Minitest::Test
+  def test_registers_offense_when_second_argument_is_not_a_string
+    assert_offense(<<~RUBY)
+      class FooTest < Minitest::Test
+        def test_do_something
+          my_list = [1, 2, 3]
+          assert(3, my_list.length)
+          ^^^^^^^^^^^^^^^^^^^^^^^^^ Did you mean to use `assert_equal(3, my_list.length)`?
+        end
+      end
+    RUBY
+  end
+
+  def test_does_not_register_offense_when_assert_is_called_with_one_argument
+    assert_no_offenses(<<~RUBY)
+      class FooTest < Minitest::Test
+        def test_do_something
+          assert(true)
+        end
+      end
+    RUBY
+  end
+
+  def test_does_not_register_offense_when_second_argument_is_a_string
+    assert_no_offenses(<<~RUBY)
+      class FooTest < Minitest::Test
+        def test_do_something
+          assert([], "empty array should be truthy")
+        end
+      end
+    RUBY
+  end
```

Comment:
Can you add the following test cases?

```ruby
  def test_does_not_register_offense_when_second_argument_is_an_interpolated_string
    assert_no_offenses(<<~'RUBY')
      class FooTest < Minitest::Test
        def test_do_something
          assert([], "empty array should be truthy #{additional_message}")
        end
      end
    RUBY
  end

  def test_does_not_register_offense_when_using_assert_equal_with_two_arguments
    assert_no_offenses(<<~'RUBY')
      class FooTest < Minitest::Test
        def test_do_something
          assert_equal([], actual)
        end
      end
    RUBY
  end
```

Reply:
Fixed in 912fd40bd0ba71fdd9a3982a7d09100013710f3f

I changed the second test a bit since `assert_equal([], actual)` should suggest `assert_empty?` (as of #118 being merged). It looks like `assert_no_offenses` won't fail if a violation for a different cop is registered but I'd feel better if the example had no violations whatsoever.

Reply:
> `assert_equal([], actual)` should suggest assert_empty?

The role of each cop force should be simple, so let `Minitest/AssertEmptyLiteral` cop make suggestions for `assert_empty?`. So the cop is up to the suggestion to `assert_equal` :-)